board.py

```diff
@@ -0,0 +1,112 @@
+
+import random
+import create_dictionary
+import time
+import sys
+
+
+def read_all_words():
+
+    """This methos is used to return words list needed to process further."""
+
+    words = []
+    text = open("//Users/darshandoshi/Desktop/boggle-master/words.txt").read()
```

Comment:
this isn't going to work for other users due to the hardcoded path.  Is it possible to make it more general?